src/pages/docs/reference/fulfillment-delegation/models/carrier/billing/billing-categories.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: BillingCategories
+description: An Enum of the possible values for category of a billing line item
+---
+
+BillingCategories
+===============================================
+An Enum of the possible values for category of a billing line item
+
+
+<Reference>  
+  <Field name="Uncategorized">
+    <Description>
+      "Uncategorized" - Not categorized 
+    </Description>
+  </Field>
+  <Field name="Shipping">
+    <Description>
+      "Shipping" - 
```

Comment:
where did this and all these values come from? 

https://connect.shipengine.com/docs/reference/billing-line-item

They don't match here and they don't have descriptions

Reply:
These are from the fulfillment delegation api model for Carrier --> Billing --> BillingCategory in your branch: https://github.com/ShipEngine/connect/blob/bradl/init-fulfillment-delegation-api/packages/fulfillment-delegation-api/src/models/carrier/billing/billing-categories.ts

Reply:
Ah I see.  Since we're making public documentation, and it's a copy of an existing documentation, we should be using the existing public documentation as a gold standard of what we want to present.  The models in the codebase don't match that, and should be used to just know which documentation we need to copy from the public documentation.